test/support/enum.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Ecto.Enum do
```

Comment:
I think we can move this to lib/ecto/enum.ex to be quite honest. It is almost ready, we just really need docs.